ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/ForkIdManager.java

```diff
@@ -257,4 +316,9 @@ private static void intToBigEndian(final int n, final byte[] bs, int off) {
     bs[++off] = (byte) (n >>> 8);
     bs[++off] = (byte) (n);
   }
+
+  @FunctionalInterface
+  private interface ForkIDChecker {
```

Comment:
I suggest we put `implements Predicate<ForkId>` here so that we can get all the other `Predicate` methods for free

Reply:
Nice catch. Done